lib/testrail.js

```diff
@@ -24,102 +23,131 @@ class TestRail {
 	async getCases(projectId, suiteId) {
 		try {
 			const res = await this.axios.get(`get_cases/${projectId}&suite_id=${suiteId}`);
+			output.log(`getCases: SUCCESS - the request data is ${JSON.stringify(data)}`);
```

Comment:
I guess we don't need this for GET request right?

Reply:
erm...maybe not, but I thought I might as well do it to keep it in line with the rest?

Reply:
my gut feeling that the data is here is null :D

Reply:
and what if I tell you that might be slightly wrong? 😓 
![image](https://user-images.githubusercontent.com/16047402/132692640-223a1701-5f1a-4653-858d-cd2f265fde58.png)

Reply:
:D I hardly see a GET call with payload